[MIG][13.0] stock_picking_batch_extended: Migration to v13.0

sergio-teruel opened the pull request:
TT21686

Commit:
[FIX] stock_picking_mass_action return backorder if any of pickings need it

AdriaGForgeFlow committed:
[12.0][FIX] stock_picking_batch_extended

pedrobaeza committed:
[MIG] stock_picking_batch_extended: Migration scripts

- Converts old stock.batch.picking in stock.picking.batch
- Restore link in stock.picking
- Other needed stuff

carlosdauden committed:
[FIX] stock_picking_batch_extended: float_is_zero required parameters

carlosdauden committed:
[IMP] stock_picking_batch_extended: UX improvements

sergio-teruel commented:
Rebase done!! After stock_picking_mass_action has been merged!

carlosdauden committed:
[IMP] stock_picking_batch_extended: Add followers, attachments, chatter and activities

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (98 of 98 strings)

Translation: stock-logistics-workflow-12.0/stock-logistics-workflow-12.0-stock_picking_batch_extended
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_picking_batch_extended/es/

Commit:
Added translation using Weblate (Spanish (Chile))

Currently translated at 98.0% (96 of 98 strings)

Translation: stock-logistics-workflow-12.0/stock-logistics-workflow-12.0-stock_picking_batch_extended
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_picking_batch_extended/es_CL/

fcolus committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (98 of 98 strings)

Translation: stock-logistics-workflow-12.0/stock-logistics-workflow-12.0-stock_picking_batch_extended
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_picking_batch_extended/pt_BR/

rafaelbn commented:
@rousseldenis please could you update labels? Thanks

alan196 reviewed (approved):
Functional review 👍  it works

pedrobaeza commented:
@sergio-teruel Travis is red

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 100.0% (98 of 98 strings)

Translation: stock-logistics-workflow-12.0/stock-logistics-workflow-12.0-stock_picking_batch_extended
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_picking_batch_extended/sl/

alan196 commented on stock_picking_batch_extended/__manifest__.py:
```diff
@@ -0,0 +1,28 @@
+    "development_status": "Mature",
+    "maintainers": ["Camptocamp", "Tecnativa"],
+    "category": "Warehouse Management",
+    "depends": ["stock_picking_batch", "delivery", "stock_picking_mass_action"],
```
Why did this module depends on delivery?

sergio-teruel replied:
Hi!! this module depends on delivery because we print carrier info from picking.
And carrier_id field is created by delivery module.

sergio-teruel committed:
[IMP] stock_picking_batch_extended: Request confirmation to cancel pickings

victoralmau commented:
@sergio-teruel Travis is red

sysadminmatmoz committed:
Translated using Weblate (Slovenian)

Currently translated at 100.0% (100 of 100 strings)

Translation: stock-logistics-workflow-12.0/stock-logistics-workflow-12.0-stock_picking_batch_extended
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-12-0/stock-logistics-workflow-12-0-stock_picking_batch_extended/sl/

pedrobaeza committed:
[IMP] stock_picking_batch_extended: black, isort, prettier

sergio-teruel committed:
[MIG] stock_picking_batch_extended: Migration to 13.0

* Standard procedure
* Fix tests

pedrobaeza reviewed (approved):
I have amended commit history and split pre-commit stuff, and code review + tests are OK

pedrobaeza commented:
/ocabot merge nobump
